compiler/Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV CARGO_TARGET_DIR="/target"
 
 RUN cargo install dylint-link cargo-dylint
 
-RUN cargo install --git https://github.com/paritytech/cargo-contract
+RUN cargo install cargo-contract
```

Comment:
Same, this should be part of the ink! 3.0 PR

Reply:
See above, then I should probably not just go back to the git repo but also fix the commit hash to a working one...